Makefile

```diff
@@ -8,3 +8,18 @@ alembic-downgrade:
 	docker compose exec api sh -c "python -m alembic downgrade -1"
 adduser:
 	docker compose exec api sh -c "python cli.py adduser $(params)"
+
+test-backend:
+	cd backend && poetry run pytest -v
+
+test-backend-w-db:
+	cd backend && ENGAGE_BACKEND_TESTS__TEST_DATABASE=true poetry run pytest -v
+
+start-test-db:
+	docker compose up -d database
+
+migrate:
+	cd backend/src && poetry run python -m alembic upgrade head
```

Comment:
True, and maybe we can find a better name for it to highlight the difference. I like it because spinning up a database in a container and applying migrations to it directly is a lot faster then building the api image. I like it if there are fast options to run unit tests as it lowers the threshold of actually doing it.

Reply:
That make sense!